lib/jekyll-remote-theme/mock_gemspec.rb

```diff
@@ -9,16 +9,43 @@ class MockGemspec
       extend Forwardable
       def_delegator :theme, :root, :full_gem_path
 
+      DEPENDENCY_PREFIX = %r!^\s*[a-z]+\.add_(?:runtime_)?dependency!
```

Comment:
Hm. I wish we could just use _config.yml, since we can load that safely. What is the advantage of using the gemspec? What if a remote theme doesn't specify a gemspec but does specify a _config.yml? Could we try both?

Reply:
I could see it either way. If you're a theme, you'd then have to list dependencies twice. That said, if you're using e.g., the page gem, you'd have to list it twice in your gemspec.

Do you think it's more intuitive to look to the plugins key of the theme? Will that cause confusion when other values aren't carried over? FWIW, [the example theme](https://jekyllrb.com/docs/themes/#creating-a-gem-based-theme) doesn't have a config file.

Reply:
If they build an example site, they’ll specify a _config.yml because Pages doesn’t parse the gemspec. While developing, they may want to use a config file instead of a gemspec for simplicity. So I’d say doing both—and this mimicking the normal jekyll experience—would be best. Reading in the theme config would be interesting in general!

Reply:
The `jekyll-data` plugin reads and loads a config file within a **theme-gem** when available.. and merges that data into the original `site.config` hash.. perhaps have something like that here too?